server/eclipse-project/script/init-db

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+readonly __dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+readonly cmd="psql -d corla -a -f "${__dir}/insert-test-creds.sql""
```

Comment:
I don't love the fact that this is a duplicate copy of the .psql from the `test` hierarchy. Means we have to keep them in sync.